README.md

```diff
@@ -6,88 +6,88 @@
 
-**This module uses APIs there are undocumented and are not covered by any**
-**deprecation policy, and this may be subject to change without notice.**
+*This module is experimental, and should be used by early adopters. This module uses APIs there may be undocumented and may be subject to change without notice.*
 
-**The API this module exports is experimental and is not covered by any**
-**deprecation policy, and may be changed without notice.**
+[Cloud Trace](https://cloud.google.com/cloud-trace/) is a feature of [Google Cloud Platform](https://cloud.google.com/) that collects latency data from your applications and displays it in near real-time in the [Google Developers Console](https://console.developers.google.com/?_ga=1.258049870.576536942.1443543237). This module provides Cloud Trace support for Node.js applications.
 
-## Quick Start
+![Cloud Trace Overview](doc/images/cloud-trace-overview-page.png)
 
-Prerequisites:
-
-1. You should already be familiar with deploying Node.js apps you to the Google Cloud Platform (GCP).  See https://cloud.google.com/nodejs/ for getting started with deploying Node.js apps on GCP.
-1. You need to have created a project on GCP. You don't necessarily have to deploy your app to GCP, but traces must be attributed to a project to be viewed.
+## Prerequisites
 1. You are using / deploying-with Node.js version 0.12 or greater.
+1. You need to have created a project on GCP. You don't necessarily have to deploy your app to GCP, but traces must be attributed to a project to be viewed.
+1. [Enable the Trace API](https://console.developers.google.com/flows/enableapi?apiid=cloudtrace) for your project.
+1. (Only if you are deploying to Google Compute Engine) Your VM instances need to be created with the 'Allow API access' checkbox selected (see screenshot). If you already have VMs that were created without API access, you will additionally need to follow the instructions in the *Running elsewhere* section.
 
-[Enable the Trace API](https://console.developers.google.com/flows/enableapi?apiid=cloudtrace&_ga=1.158570170.161926390.1438300999) for your project.
+![GCE API](doc/images/gce.png?raw=true)
 
-Install the trace agent module:
-```
-npm install --save @google/cloud-trace
-```
+## Running on Google Cloud Platform
+1. Install module, save it as a dependency in your `package.json`:
 
-Load the module as the very first thing that gets done at the startup of your application:
+   `npm install --save @google/cloud-trace`
+1. Ensure that the trace agent gets loaded **as the first thing** during your application's startup. The best way to do this is via the `--require` command line option available in Node.js v4. You should add this option to the startup script section in your package.json.
 
-```javascript
-// Activate Google cloud trace agent (*must be first line of the app*)
-require('@google/cloud-trace').start();
-```
+   ```
+   ...
+   "scripts" {
+     "start": "node --require @google/cloud-debug server.js"
```

Comment:
Actually, require does not work for trace. The require does nothing until the agent is started.